torchrl/envs/libs/dm_control.py

```diff
@@ -240,9 +248,8 @@ def _output_transform(
             timestep_tuple = (timestep_tuple,)
         reward = timestep_tuple[0].reward
 
-        done = False  # dm_control envs are non-terminating
+        truncated = terminated = done = False  # dm_control envs are non-terminating
```

Comment:
```suggestion
        truncated = terminated = gym_done = False  # dm_control envs are non-terminating
```

I spotted this here, but I'm probably missing it in some other places. Feel free to ignore it if it's not essential to change it everywhere.

Reply:
I think it would be a good thing to always call the old gym done `gym_done`

Reply:
Or we could just stick to torchrl's names? Done is completed, stop is eot, truncated is interrupted, always. No terminated for instance.
"Gym_done" could also be twisted (if I'm a picky devil's advocate) into "done but the gym version which is terminated"

Reply:
Ah would be even better.
So right after a a gym call have
```python
# new gym.
obs, reward, done, truncated, info = env.step(action) 
# old gym
obs, reward, stop, info = env.step(action)
```
and not have any `terminated` variable in the code?

Reply:
that's the idea!